.github/workflows/kata-deploy-test.yaml

```diff
@@ -48,12 +48,19 @@ jobs:
           - rootfs-initrd
           - shim-v2
     steps:
+      - name: resolve pr refs
```

Comment:
Could you add a comment that quickly explains why it is done?  Some little help for us in the future  to help us to understand why we add it and when we could remove it.

Something like:
```sh
# The Github action event `issue_comment` does not provide the right ref (commit/branch) from the PR to be tested.
# It is needed to use this third party action until this is issue is fixed
# URL_TO_ISSUE
```

Reply:
Partially done, thanks!
There's no issue to link, as this seems to be intented behaviour of the `issue_comment` event.